apply-labels/src/user/oauth.py

```diff
@@ -0,0 +1,37 @@
+from http import HTTPStatus
+
+from src.helpers import AdminRESTClient, loggable_debug
+
+_USER_PREFIX = "/api/v1/users"
+
+
+class UserOAuthCredentials:
+    def __init__(self, admin_client: AdminRESTClient):
+        self.client = admin_client
+        self.__check_access()
+
+    @loggable_debug
+    def generate_oauth_credentials(self, emails: list[str]):
```

Comment:
@ivanm-gdp udah coba pakai 1 akun aja belom? atau bisa pake example nya (`labelers.example.json`) kan itu yg belom punya credentials cuma 1 akun, nah di aku kena error yg `emails must be an array"`

Reply:
hoh 👀 

hm

aku tadi nge-postman sih 1 email dan 2 email bisa

let me see requests.post ini ngapain aja 🤔

Reply:
iya pas aku review pr mas rudi juga bisa si pake 1 email

Reply:
ini udah fixed ya :+1: